### ISSUE TITLE
Address missing alt text

### ISSUE THREAD

**Issue by nickpistone** [GSA/px-benefit-finder#1033]
### Description

Decide if the following images need alt text and if so, add alt text. Assigning to Angelina so that she can ensure that the alt text follows 508 guidelines and to Gene so that he can make edits as necessary. This issue was flagged on all 4 of the English BF pages including the landing page, but not the Spanish pages. We should verify that whatever solution we come up with is implemented on the Spanish pages as well. 

[https://www.usa.gov/themes/custom/usagov/assets/img/icon-dot-gov.svg](https://www.usa.gov/themes/custom/usagov/assets/img/icon-dot-gov.svg)
[https://www.usa.gov/themes/custom/usagov/assets/img/icon-https.svg](https://www.usa.gov/themes/custom/usagov/assets/img/icon-https.svg)
[https://www.usa.gov/themes/custom/usagov/images/favicon-57.png](https://www.usa.gov/themes/custom/usagov/images/favicon-57.png)

### User Story

#1027

**Comment by scottqueen-bixal** [comment]
these images come from USAGov code and are part of the global theme, please share this with USAGov team to see if they are intending on fixing.

https://github.com/usagov/usagov-2021/tree/c3a776fbb54868b4e40af10f5b41f8f3a07f9482/web/themes/custom/usagov/images

**Comment by nickpistone** [comment]
Shared this with David Kaufmann
